session_0/fizz_buzz.rb

```diff
@@ -37,5 +37,21 @@
 # We will raise an `ArgumentError` exception to let the caller know that
 # their function arguments were incorrect.
 def fizz_buzz(n:, x: 3, y: 5)
-  raise NotImplementedError # TODO
+  raise ArgumentError if n<0||x<=0||y<=0 
+  
+  l=[]
+  for i in 1..n
```

Comment:
Can map function be used instead?

https://www.rubyguides.com/2018/10/ruby-map-method/